smartsim/entity/model.py

```diff
@@ -46,6 +46,24 @@
 logger = get_logger(__name__)
 
 
+def _read_model_parameters(params_dict: t.Dict[str, t.Any]) -> t.Dict[str, str]:
```

Comment:
Call this `_parse_model_parameters` or `_set_model_parameters`?

Reply:
changed to parse